google-cloud-clients/google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/EnhancedBigtableStubSettings.java

```diff
@@ -420,14 +422,14 @@ private Builder() {
       // Per-method settings using baseSettings for defaults.
       readRowsSettings = ServerStreamingCallSettings.newBuilder();
       readRowsSettings
-          .setRetryableCodes(baseDefaults.readRowsSettings().getRetryableCodes())
```

Comment:
We should just add ABORTED to gapic codes:

```java
// Allow retrying ABORTED statuses. These will be returned by the server when the client is too slow to read the rows.
// This makes sense for the java client because retries happen after the row merging logic. Which means that the retry
// will not be invoked until the current buffered chunks are consumed. 
readRowsSettings.setRetryableCodes(
  ImmutableSet.builder()
   .addAll(baseDefaults.readRowsSettings().getRetryableCodes())
   .add(baseDefaults.readRowsSettings().getRetryableCodes())
   .build());
```

Reply:
By any chance do you mean something like this:
```java
readRowsSettings.setRetryableCodes(
  ImmutableSet.builder()
   .addAll(baseDefaults.readRowsSettings().getRetryableCodes())
   .add(Code.ABORTED)
   .build());
```

Reply:
Oops , sorry about that. Yes that’s what I meant